components/header/VotingWeight.tsx

```diff
@@ -2,7 +2,7 @@ import { Flex, Text } from 'theme-ui';
 import useSWR from 'swr';
 import useAccountsStore from 'stores/accounts';
 import getMaker from 'lib/maker';
-import { getVotingWeightCopy } from 'lib/polling/getVotingWeightCopy';
+import { getVotingWeightCopy } from 'modules/polls/lib/getVotingWeightCopy';
```

Comment:
I don't have very strong feelings about this but I feel like `utils` or `helpers` could be a better directory name for `lib` within these modules. The functions inside seem mostly to be utility/helper functions. Any thoughts?